docs/advanced_stardoc_usage.md

```diff
@@ -0,0 +1,87 @@
+<nav class="toc">
+  <h2>Contents</h2>
+  <ul>
+    <li><a href="#docstring-formatting">Docstring Formatting</a></li>
+    <li><a href="#custom-output">Custom Output</a></li>
+  </ul>
+</nav>
+
+This document covers a number of advanced topics pertaining to using Stardoc.
+
+
+<a name="docstring-formatting"></a>
+## Docstring Formatting
+
+You may want to inline various kinds of formatting in the docstrings adjacent
+to your Starlark code. Use typical markdown formatting constructs instead of
+HTML tags.
+
+For example:
+```python
+def my_function(foo, bar):
+  """Does some cool stuff.
+
+  Oh, by the way, have you heard about [Stardoc](https://github.com/bazelbuild/stardoc)?
+
+  Args:
+    foo: You don't know what a **foo** is?
+    bar: Two variables, `x` and `y`, walk in to a bar...
+  """
+  ...
+```
+
+Markdown formatting constructs are handled appropriately by Stardoc's default
+output format ("markdown_tables"), even as part of a table.
+
+
+<a name="custom-output"></a>
+## Custom Output
+
+Stardoc's output format is customizable; while Stardoc's output is markdown
+by default, you may define a different output format for your documentation.
+
+Customization is done at the level of "output templates". To customize the
+doc output for a particular type of Starlark definition (such as a "rule" or a
+"function"), you will need to:
+
+1. Create a new custom output template to describe how this type of object should
+   be rendered.
+2. In your `stardoc()` target, set the matching "_template" attribute to point to
```

Comment:
```suggestion
2. In your `stardoc()` target, set the matching `_template` attribute to point to
```

Reply:
the stray underscore italicizes the rest of the document in review mode.

Reply:
Oof. Thanks a bunch.